app/view/grid/ExampleGrid.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Example classes for a CpsiMapview.view.grid.Grid grid
+ */
+
+// define the model
+Ext.define('CpsiMapview.model.GridExample', {
+    requires: 'GeoExt.data.model.Feature',
+    extend: 'GeoExt.data.model.Feature',
+    idProperty: 'EquipmentId',
+    fields: [
+        {
+            name: 'UnitTypeName',
+            type: 'string'
+        },
+        {
+            name: 'EquipmentId',
+            type: 'integer'
+        }
+    ]
+});
+
+// create the store
+Ext.define('CpsiMapview.store.GridExample', {
+    extend: 'CpsiMapview.store.WfsFeatures',
+    alias: 'store.GridExample',
+    url: 'https://plmonaghandev.compass.ie/mapserver/?',
+    storeId: 'GridExample',
+    model: 'CpsiMapview.model.GridExample',
+    typeName: 'LightUnit',
+    layerOptions: {
+        displayInLayerSwitcher: false,
+        name: 'GridExampleLayer'
+    },
+    sorters: [{
+        property: 'EquipmentId',
+        direction: 'ASC'
+    }]
+});
+
+// create the view model
+Ext.define('CpsiMapview.view.grid.ExampleGridModel', {
+    extend: 'CpsiMapview.model.grid.Grid',
+    alias: 'viewmodel.example_grid',
+    data: {
+        vectorLayerKey: 'LIGHT_UNIT_MVT',
+        gridStoreType: 'GridExample',
+        gridLayerName: 'GridExampleLayer' // TODO this is duplicated in layerOptions above
+    }
+});
+
+// create the grid view
+Ext.define('CpsiMapview.view.grid.ExampleGrid', {
+    extend: 'CpsiMapview.view.grid.Grid',
+    xtype: 'examplegrid',
```

Comment:
A `cmv_` would be good here to keep the xtypes consistent

Reply:
Updated